monitoring/storage.go

```diff
@@ -50,17 +88,29 @@ type HighWatermarkCheckerData struct {
 	AvailableBytes uint64 `json:"available_bytes"`
 }
 
-// FailureMessage returns failure watermark check message
-func (d HighWatermarkCheckerData) FailureMessage() string {
-	return fmt.Sprintf("disk utilization on %s exceeds %v percent (%s is available out of %s), see https://gravitational.com/telekube/docs/cluster/#garbage-collection",
+// WarningMessage returns warning watermark check message
+func (d HighWatermarkCheckerData) WarningMessage() string {
+	return fmt.Sprintf("disk utilization on %s exceeds %v%% (%s is available out of %s), cluster will degrade if usage exceeds %v%%, see https://gravitational.com/gravity/docs/cluster/#garbage-collection",
```

Comment:
Hmm, I'm a bit on the fence on how much value it ads to include the output about what point the cluster will degrade at.

Reply:
Since this is just a user message we can leave it for now and change it later on if necessary. @a-palchikov suggested we include the point the cluster will degrade, so I think it adds some value to the user experience.